icingaweb2/icingaweb2.spec

```diff
@@ -412,7 +412,10 @@ fi
 %{basedir}/library/vendor/Zend
 
 %changelog
-* Mon Nov 11 2017 Markus Frosch <markus.frosch@icinga.com> 2.5.0-1
+* Tue Nov 28 2017 Eric Lippmann <eric.lippmann@icinga.com> 2.5.0-1
```

Comment:
Version should be 2.5.0-2

Reply:
Only if we released 2.5.0-1 already, so this would be fine.

Reply:
Sorry for the noise.